app/events/services/service_deleted_event.rb

```diff
@@ -5,7 +5,6 @@ def self.create(service)
     new(
       service_id: service.id,
       service_name: service.name,
-      service_created_at: service.created_at.to_s,
```

Comment:
Has this never been used?

Reply:
I see it was introduced here: https://github.com/3scale/porta/commit/6dcc86912d4ecbdf0ab2dc6718c2e0510ca38b56

Reply:
Ok. I see it now. It's `ThreeScale::Core::Service.delete_stats` that now only requires `service_id` and no detailed payload anymore.

Reply:
@guicassolato https://github.com/3scale/porta/pull/1826#issuecomment-620476336